components/CookieBanner.vue

```diff
@@ -5,23 +5,25 @@
         class="notices flex overflow-hidden pointer-events-none fixed p-[2em] inset-0 is-bottom items-center w-full z-[1000]">
         <div
           role="alertdialog"
-          class="snackbar is-success is-bottom-left py-2 px-4 mb-3">
-          <div>
+          class="snackbar is-success is-bottom-left !py-4 md:!py-2 px-4 mb-3 flex-col md:flex-row">
+          <div class="md:flex md:gap-1">
             {{ $t('cookies.notice') }}, see
             <nuxt-link to="/cookie-notice" target="_blank" class="is-underlined"
               >Cookie Policy</nuxt-link
             >
             for details.
           </div>
-          <div class="bar ml-4 mr-4" />
-          <div class="action is-success">
+          <div
+            class="bg-separator-line-color ml-4 mr-4 my-4 md:!my-0 !w-full !h-[1px] md:!w-[1px] md:!h-[13px]" />
```

Comment:
`ml-4 mr-4 my-4` m-4 ?

checked and  the important here`md:!my-0` seems to be needed.

but the others `w-full h-[1px] md:w-[1px] md:!h-[13px]` seem to work as expect without `!` 

pls remove any important if they are not needed.

Reply:
> `ml-4 mr-4 my-4` m-4 ?
> 
updated


> checked and the important here`md:!my-0` seems to be needed.
> 
> but the others `w-full h-[1px] md:w-[1px] md:h-[13px]` seem to work as expect without `!`
> 
> pls remove any important if they are not needed.

It's not needed if there is only tailwind, but now there are some side effects from Bulma styles..

Reply:
> > `ml-4 mr-4 my-4` m-4 ?
> 
> updated
> 
> > checked and the important here`md:!my-0` seems to be needed.
> > but the others `w-full h-[1px] md:w-[1px] md:h-[13px]` seem to work as expect without `!`
> > pls remove any important if they are not needed.
> 
> It's not needed if there is only tailwind, but now there are some side effects from Bulma styles.. It dose not work as expected without `!`

yep I'm aware of that, it's pretty annoying, I've encountered those pretty often when adding any  `margin` or `padding`   classes , it forces you to use `!` otherwise it ends up using the bulma class.